pqos/alloc.c

```diff
@@ -379,21 +387,32 @@ set_mba_cos(const unsigned class_id,
          * Set all selected classes
          */
         for (i = 0; i < sock_num; i++) {
+                const char *unit;
+                const char *package;
                 int ret = pqos_mba_set(sock_ids[i], 1, &mba, &actual);
 
+                if (cpu->vendor == PQOS_VENDOR_AMD) {
+                        package = "Core Complex ";
+                        unit = "";
+                } else {
+                        package = "SOCKET ";
```

Comment:
Please remove redundant space
```suggestion
                        package = "SOCKET";
```